openapi/doc/index.rst

```diff
@@ -67,10 +73,25 @@ via Database Structure Menu (only for developers)
     
     * TODO: put this in the form view::
 
-          Format for Fields Set is 
+          Format for Fields Set is a python tuple
+          * many2one: nested tuple
+          * ***2many: nested list with tuple in it ([])
+          # many2one fields:
+          ('invoice_id', (
+              'id',
+              'number',
+          )),
+          # many2many fields:
+          ('invoice_line_tax_ids', [(
+              'id',
+              'name',
+              'type',
+              'amount',
+              'price_include',
+          )]),
```

Comment:
to discuss. 
For me, it's just difficult to type all those parenthesises, commas, quotes,

Reply:
This format should be easy to type
```
name
invoice_id/id
invoice_id/name
invoice_line_tax_ids/id
invoice_line_tax_ids/name
invoice_line_tax_ids/type
```

Reply:
I agree with the proposed format for manual specs.
We should parse ***and valdiate*** it on `write`, though.
So that the parsing logic is as minimal as possible during a request...